cfgov/v1/migrations/0226_update_enforcement_action_data.py

```diff
@@ -0,0 +1,31 @@
+# Generated by Django 2.2.13 on 2020-07-13 17:19
+from django.db import migrations
+
+
+def forward(apps, schema_editor):
+    EnforcementActionStatus = apps.get_model('v1', 'EnforcementActionStatus')
+    for existing, updated in (
+        ('Expired/Terminated/Dismissed', 'expired-terminated-dismissed'),
+        ('Pending Litigation', 'pending-litigation'),
+        ('Post Order/Post Judgment', 'post-order-post-judgment'),
+    ):
```

Comment:
This is probably unnecessary, but I like the idea of importing the list of statuses & labels from `v1/util/ref.py` and flipping the order in this for loop to `updated, existing`. That was we aren't duplicating the raw data/it's impossible to get bitten by a typo here (even though it looks typo-free to me).

Reply:
Theoretically, if we want this data migration to continue to work correctly in the future, we would need to *not* refer to `v1/util/ref.py`, since that file might change.

I feel pretty confident there are no typos and all records get updated correctly. When I tested locally, the total number of `EnforcementActionStatus` objects in the database was equal to the number of results that come up when searching for `expired-terminated-dismissed`, `pending-litigation`, and `post-order-post-judgment`.

```
cfgov=> select count( distinct action_id ) from v1_enforcementactionstatus;
 count
-------
   245
```

and

<img width="834" alt="Screen Shot 2020-07-14 at 10 37 12 AM" src="https://user-images.githubusercontent.com/4295388/87439228-868bfa00-c5be-11ea-89a0-ad5b03b8eb7d.png">